docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -202,6 +202,10 @@ class DialogPage extends React.Component {
 
   }
 
+  _onDialogSubmit() {
```

Comment:
This handler was absent - I guess it should be here, performing some dummy action.